homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -55,6 +55,21 @@
 OAUTH_TOKEN_TIMEOUT_SEC = 30
 
 
+@callback
+def async_get_redirect_uri(hass: HomeAssistant) -> str:
+    """Return the redirect uri."""
+    if "my" in hass.config.components:
+        return MY_AUTH_CALLBACK_PATH
```

Comment:
It seems there are no existing tests for all cases in the code. We should add that.

Reply:
I'll do it in a follow up PR since I'd like to merge this for beta.

Reply:
https://github.com/home-assistant/core/pull/136976